tests/test_italy_stationary_example.py

```diff
@@ -0,0 +1,10 @@
+import calliope_pathways
+
+
+def test_solve_model():
+    """Quick build test of the static italy model using extreme resampling."""
+    overrides = {"config.solve.solver": "cbc",
+                 "config.init.time_resample": "8760h"}
```

Comment:
You could build with a monthly resolution "1m"?

Reply:
Fixed!